### PR TITLE
[Update] Remove all Classic Manager guides from the library

### PR DESCRIPTION
**Pull request by leslitagordita**
* removes all guides with `-classic-manager` appended
* Adds aliases from classic to cloud guide
* Removes `classic_manager_link` front matter from any cloud guides

### CODE REVIEW

File: docs/networking/dns/configure-your-linode-for-reverse-dns/index.md [hunk 1]
```diff
@@ -5,13 +5,13 @@ author:
 description: 'Reverse DNS (rDNS) resolves an IP address to the designated domain name. This guide will teach you how to set it up.'
 keywords: ["reverse", "dns", "PTR"]
 license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
-aliases: ['networking/Setting-Up-Reverse-DNS-Lookup/','dns-guides/configuring-dns-with-the-linode-manager/','networking/configure-your-linode-for-reverse-dns/','networking/dns/setting-reverse-dns/','dns-manager/']
+aliases: ['networking/dns/configure-your-linode-for-reverse-dns-classic-manager-classic-manager/','dns-manager/','networking/configure-your-linode-for-reverse-dns/','networking/dns/setting-reverse-dns/','dns-guides/configuring-dns-with-the-linode-manager/','networking/setting-up-reverse-dns-lookup/']
```
Comment (nmelehan) [inline docs/networking/dns/configure-your-linode-for-reverse-dns/index.md]: got a double `classic-manager` here:

`networking/dns/configure-your-linode-for-reverse-dns-classic-manager-classic-manager`

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 6ed948a995dfc529c9e5c8e7894a3f64c85d8461

https://deploy-preview-3098--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/networking/dns/configure-your-linode-for-reverse-dns/index.md (hunk 1 above)
Comment (leslitagordita) [reply docs/networking/dns/configure-your-linode-for-reverse-dns/index.md]: fixed

**Comment by leslitagordita** [comment]
See: https://github.com/linode/docs/pull/3100

**Pull request closed without merging**